app/src/util/form-schema/manual-input-hierarchy.json

```diff
@@ -1490,6 +1560,835 @@
+        ],
+        "suggestedActivities": [
+          {
+            "id": "fuel-sales-aviation-activity",
+            "aviation-vehicle-type": "vehicle-type-all"
+          },
+          {
+            "id": "fuel-sales-aviation-activity",
+            "aviation-vehicle-type": "vehicle-type-passenger-aircraft"
+          },
+          {
+            "id": "fuel-sales-aviation-activity",
+            "aviation-vehicle-type": "vehicle-type-cargo-aircraft"
+          }
+        ]
+      },
+      {
+        "id": "geographic-aviation-methodology",
+        "inputRequired": [
+          "total-distance-travelled-in-in-boundary-trips",
+          "fuel-intensity"
+        ],
+        "activities": [
+          {
+            "id": "geographic-aviation-activity",
+            "unique-by": ["aviation-vehicle-type", "aviation-fuel-type"],
+            "activity-title": "total-distance-travelled-in-boundary",
+            "units": ["units-kilometers", "units-miles"],
+            "extra-fields": [
+              {
+                "id": "aviation-vehicle-type",
+                "options": [
+                  "vehicle-type-all",
+                  "vehicle-type-passenger-aircraft",
+                  "vehicle-type-cargo-aircraft",
+                  "vehicle-type-private-and-business-aircraft",
+                  "vehicle-type-helicopters"
+                ]
+              },
+              {
+                "id": "aviation-fuel-type",
+                "options": [
+                  "fuel-type-gasoline",
+                  "fuel-type-diesel",
+                  "fuel-type-lpg",
+                  "fuel-type-cng",
+                  "fuel-type-biofuel"
+                ]
+              },
+              {
+                "id": "aviation-fuel-intensity",
+                "type": "number",
+                "units": [
+                  "units-grams-per-kilometer",
+                  "units-kilograms-per-kilometer"
+                ]
+              }
+            ]
+          }
+        ],
+        "suggestedActivities": [
+          {
+            "id": "geographic-aviation-activity",
+            "aviation-vehicle-type": "vehicle-type-all"
+          },
+          {
+            "id": "geographic-aviation-activity",
+            "aviation-vehicle-type": "vehicle-type-passenger-aircraft"
+          },
+          {
+            "id": "geographic-aviation-activity",
+            "aviation-vehicle-type": "vehicle-type-cargo-aircraft"
+          }
+        ]
+      }
+    ],
+    "directMeasure": {
+      "id": "aviation-direct-measure-methodology",
+      "extra-fields": [
+        {
+          "id": "aviation-fuel-type",
+          "options": [
+            "fuel-type-gasoline",
+            "fuel-type-diesel",
+            "fuel-type-lpg",
+            "fuel-type-cng",
+            "fuel-type-biofuel"
+          ]
+        },
+        {
+          "id": "aviation-vehicle-type",
+          "options": [
+            "vehicle-type-passenger-aircraft",
```

Comment:
missing options:
the option: "vehicle-type-all"
"exclusive": "vehicle-type-all"